hqdm-rdf/src/main/java/uk/gov/gchq/hqdm/rdf/Triples.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2021 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package uk.gov.gchq.hqdm.rdf;
+
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+import uk.gov.gchq.hqdm.model.Thing;
+import uk.gov.gchq.hqdm.rdf.iri.IRI;
+
+/**
+ * Convert Things to Triple strings.
+ */
+public abstract class Triples {
```

Comment:
This is currently used only by MagmaCore for dumping data to Turtle format, and it's here since it's RDF-specific. I expect it will be needed for some future data export feature where a subset of the database content will be exported, or maybe logged, routinely. Ad-hoc full data exports would probably use the database features directly. It could be removed, but it contains some TTL formatting logic that would be useful to retain.